lib/resources/dstu2/condition.yaml

```diff
@@ -149,6 +149,26 @@ fields:
   type: code
   description: The clinical status of the condition.
   example: '{"clinicalStatus": "resolved"}'
+  action: update
+  note: |
+    <ul>
+      <li> When the category is <code>encounter-diagnosis</code>, only the <code>active</code> code is supported. </li>
```

Comment:
hmm...maybe we should combine the 1st and 2nd point.

a clinicalStatus with the value of `active` must be provided, unless the verification...

Reply:
addressed and replaced with new evidence.